tests/roles/run_tests/vars/main.yml

```diff
@@ -14,7 +14,8 @@ CLUSTER_APIENDPOINT_IP: "{{ lookup('env', 'CLUSTER_APIENDPOINT_IP') }}"
 CLUSTER_NAME: "{{ lookup('env', 'CLUSTER_NAME') | default('test1', true) }}"
 NUM_OF_CONTROLPLANE_REPLICAS: "{{ lookup('env', 'NUM_OF_CONTROLPLANE_REPLICAS') | default(1, true) }}"
 NUM_OF_WORKER_REPLICAS: "{{ lookup('env', 'NUM_OF_WORKER_REPLICAS') | default(1, true) }}"
-NUM_NODES: "{{ lookup('env', 'NUM_NODES') | default(2, true) }}"
+NUM_NODES: "{{ lookup('env', 'NUM_NODES') | default(1, true) }}"
+NUM_REDFISHVMNODES: "{{ lookup('env', 'NUM_REDFISHVMNODES') | default(1, true) }}"
```

Comment:
the defaults here aren't matching with lib/common?

Reply:
fixed